baseline-cli/src/test/kotlin/BaselineCliTest.kt

```diff
@@ -135,10 +135,11 @@ class BaselineCliTest {
     @Test
     fun `test include absent false`() {
         // Act
-        assertDoesNotThrow { BaselineCli.process(BaselineOptions(emptySarif, sarif, includeAbsent = false), stdout::append, stderr::append) }
+        assertDoesNotThrow { BaselineCli.process(BaselineOptions(Path("src/test/resources/single.sarif.json").toString(), sarif, includeAbsent = false), stdout::append, stderr::append) }
```

Comment:
```suggestion
        assertDoesNotThrow { BaselineCli.process(BaselineOptions(copySarifFromResources("src/test/resources/single.sarif.json").toString(), sarif, includeAbsent = false), stdout::append, stderr::append) }
```

Haven't checked whether this test actually does mutate, but let's be safe anyway

Reply:
Good catch, it does!